### ISSUE TITLE
testimage: xcf tests fail (used with SDL3)

### ISSUE THREAD

**Issue by madebr** [libsdl-org/SDL_image#327]
Excerpt from `testimage` output:
```
INFO:  01/06/23 14:11:49: Testing ability to load format XCF
INFO:  01/06/23 14:11:49: Assert 'Building filename should succeed ()': Passed
INFO:  01/06/23 14:11:49: Assert 'Building ref filename should succeed ()': Passed
INFO:  01/06/23 14:11:49: Assert 'Loading reference should succeed ()': Passed
INFO:  01/06/23 14:11:49: Assert 'Load /home/maarten/programming/SDL_image/test/sample.xcf ()': Passed
INFO:  01/06/23 14:11:49: Assert 'Expected width 23 px, got 23': Passed
INFO:  01/06/23 14:11:49: Assert 'Expected height 42 px, got 42': Passed
INFO:  01/06/23 14:11:49: Assert 'Converting to RGBA should succeed ()': Passed
INFO:  01/06/23 14:11:49: Assert 'Converting to RGBA should succeed ()': Passed
ERROR: 01/06/23 14:11:49: Comparison of pixels with allowable error of 0 failed 966 times.
ERROR: 01/06/23 14:11:49: First detected occurrence at position 0,0 with a squared RGB-difference of 63509.
ERROR: 01/06/23 14:11:49: Surfaces from failed comparison saved as 'CompareSurfaces0061_TestOutput.bmp' and 'CompareSurfaces0061_Reference.bmp'
ERROR: 01/06/23 14:11:49: Assert 'Surface differed from reference by at most 0 in 966 pixels': Failed
INFO: /home/maarten/programming/SDL_image/test/sample.xcf:
INFO:   Pixels:
INFO:     
INFO: 00
INFO: 00
INFO: 00
INFO: 00 
INFO: 00
...

```

The output is all-zero.

| `CompareSurfaces0061_TestOutput.bmp`  | `CompareSurfaces0061_Reference.bmp` |
| ------------- | ------------- |
|![CompareSurfaces0061_TestOutput](https://user-images.githubusercontent.com/4138939/211019255-bdfd4626-6865-4069-96dd-1c478b99d7dd.jpg) | ![CompareSurfaces0061_Reference](https://user-images.githubusercontent.com/4138939/211019285-5593dbfe-a5c7-4b47-b4c1-a94fd2110739.jpg) |

**Comment by slouken** [comment]
I can take a look at this, I probably broke it during the conversion to SDL3.

**Comment by smcv** [comment]
This seems to have been fixed, presumably by 698f2cf4.

**Comment by madebr** [comment]
That's right. Thanks!
